### PR TITLE
[test-only] apiTest. Remove group from space test

### PR DESCRIPTION
**Pull request by ScharfViktor**
## Description
Tests for removing a group from a space

## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSpacesShares/shareSpaces.feature [hunk 1]
```diff
@@ -188,3 +188,44 @@ Feature: Share spaces
...
+
+  Scenario: Users don't have access to the space if the group has been deleted
+    Given group "group2" has been created
+    And the administrator has added a user "Brian" to the group "group2" using GraphApi
+    And the administrator has added a user "Bob" to the group "group2" using GraphApi
+    And user "Alice" has shared a space "share space" to group "group2" with role "editor"
+    When the administrator deletes group "group2" using the Graph API
+    Then the HTTP status code should be "204"
+    And the user "Brian" should not have a space called "share space"
+    But the user "Bob" should not have a space called "share space"
```
Comment (individual-it) [inline tests/acceptance/features/apiSpacesShares/shareSpaces.feature:231]:
```suggestion
    And the user "Bob" should not have a space called "share space"
```

**Review by individual-it — CHANGES_REQUESTED** [review]
CI fails in the clone phase
I guess this PR needs a rebase

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>localApiTests-apiSpacesShares-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/18079/46/1) failed. Further test are cancelled...

**Comment by ScharfViktor** [comment]
in stable-2.0 there is no code to "delete a shared resource", so I switched `base:master`

I created a new PR #5365 so as not to do a big rebase

**Pull request closed without merging**